api/controllers/pr/index.js

```diff
@@ -28,52 +28,43 @@ exports.index = (req, res) => {
   ])
     .then(([prs, mrs, user, gitlab_user]) => {
       let isGitHubUser = true;
-      if (user == null) isGitHubUser = false;
       let isGitLabUser = true;
+
+      if (JSON.stringify(user) == JSON.stringify([])) isGitHubUser = false;
```

Comment:
Do we need to json stringify? Can we just `if (user === [])` ?

Reply:
no this does not work in javascript. Because == (and ===) test to see if two objects are the same object and not if they are identical objects.

user and [] are two different objects.

Reply:
It will be `[]` here: https://github.com/jenkoian/hacktoberfest-checker/pull/592/files#diff-9d1d60f8d642ee863fa2a01bfec91b5440c84bbd481945f1766893c909b7cec3R174 though right? So if we're trying to determine if that github user has returned an error `[] === []` should do it? Or even `user.length` ?

Reply:
[] === [] return false always. You canc check in the console.

Reply:
I can use user.length

Reply:
> [] === [] return false always. You canc check in the console.
Yeah, I mean if `user` if an empty array, i.e. falls into that error condition.